lib/ui/text.dart

```diff
@@ -1353,9 +1353,10 @@ class Paragraph extends NativeFieldWrapperClass2 {
     assert(boxWidthStyle != null);
     return _getBoxesForRange(start, end, boxHeightStyle.index, boxWidthStyle.index);
   }
-
   List<TextBox> _getBoxesForRange(int start, int end, int boxHeightStyle, int boxWidthStyle) native 'Paragraph_getRectsForRange';
 
+  List<TextBox> getBoxesForWidgets() native 'Paragraph_getRectsForWidgets';
```

Comment:
This is missing documentation. 

Docs should be clear about what the origin is for the coordinates in the returned list.

Reply:
also mention that calling this is expensive

Reply:
The boxes are defined by left top right bottom, so the concept of "origin" doesn't really apply, as any origin can be extracted from the definition.

Will add docs though!